### PR TITLE
ui: add support NodePort and Loadbalancer service types

### PR DESCRIPTION
**Pull request by maany**

### CODE REVIEW

File: charts/rucio-ui/templates/service.yaml [hunk 1]
```diff
@@ -13,16 +13,28 @@ metadata:
...
+  loadBalancerIP: {{ .Values.service.loadBalancerIP }}
+{{- end }}
+{{- if .Values.service.loadBalancerClass }}
+  loadBalancerClass: {{ .Values.service.loadBalancerClass }}
+{{- end }}
+{{- if .Values.service.externalTrafficPolicy }}
+  externalTrafficPolicy: {{ .Values.service.externalTrafficPolicy }}
+{{- end }}
+{{- if not .Values.service.allocateLoadBalancerNodePorts }}
+  allocateLoadBalancerNodePorts: {{ .Values.service.allocateLoadBalancerNodePorts }}
```
Comment (rdimaio) [inline charts/rucio-ui/templates/service.yaml:38]: should this be `if` instead of `if not`?

File: charts/rucio-ui/values.yaml [hunk 2]
```diff
@@ -11,13 +11,25 @@ useDeprecatedImplicitSecrets: true
 exposeErrorLogs: true
 
 service:
-  type: NodePort
   # Run the webui server on port 443 instead of 80 and accept X509 certificates and proxies
```
Comment (rdimaio) [inline charts/rucio-ui/values.yaml:14]:
```suggestion
```
This comment is out of date now

File: charts/rucio-ui/templates/service.yaml (hunk 1 above)
Comment (maany) [reply charts/rucio-ui/templates/service.yaml]: good catch ;)

File: charts/rucio-ui/values.yaml (hunk 2 above)
Comment (maany) [reply charts/rucio-ui/values.yaml:14]: Not sure what changed. 443 is still required if you need to use x509 auth :)

**Review by rdimaio — APPROVED** [review]

**Pull request merged**